xml/System.Net.Security/ServerCertificateSelectionCallback.xml

```diff
@@ -29,10 +29,11 @@
     <ReturnType>System.Security.Cryptography.X509Certificates.X509Certificate</ReturnType>
   </ReturnValue>
   <Docs>
-    <param name="sender">To be added.</param>
-    <param name="hostName">To be added.</param>
-    <summary>To be added.</summary>
-    <returns>To be added.</returns>
-    <remarks>To be added.</remarks>
+    <param name="sender">An object that contains state information for this validation.</param>
```

Comment:
The sender is SslStream itself. I think it is not strongly typed for historical reasons.

Reply:
fixed